Pkgfile

```diff
@@ -3,18 +3,18 @@
 format: v1alpha2
 
 vars:
-  TOOLS_IMAGE: ghcr.io/siderolabs/tools:v1.5.0-alpha.0-3-g81b09a5
+  TOOLS_IMAGE: ghcr.io/siderolabs/tools:v1.5.0-alpha.0-4-geedc294
 
   # renovate: datasource=github-releases depName=containernetworking/plugins
-  cni_version: v1.2.0
-  cni_sha256: f3496ddda9c7770a0b695b67ae7ee80a4ee331ac2745af4830054b81627f79b7
-  cni_sha512: fb6fb4f46ac1610b3721f5f3a6ddfb096cbf2e5d5b792306edca5351a3944d2f802170d83e5adec01420395bf64fc8a174ede61ac9b93b5ac6b938a4b48651e6
+  cni_version: v1.3.0
+  cni_sha256: f9871b9f6ccb51d2b264532e96521e44f926928f91434b56ce135c95becf2901
+  cni_sha512: 87e186b3cd64f66280f5b2293dcdd1fc22cb8f51a248124fb622adc48a893348419ba4c29c4769dede4d9e60f2e9fea5d4198f10badb4ecd20a1551e0b344e10
 
   # renovate: datasource=github-tags depName=containerd/containerd
-  containerd_version: v1.6.21
-  containerd_ref: 3dce8eb055cbb6872793272b4f20ed16117344f8
-  containerd_sha256: 9452e95455d03a00d78ae0587595d0c18555bae7912068269efa25a724efe713
-  containerd_sha512: a94ed8b8b9e153c9dc370228b659fcba1b03b3c47c8b5fcb001bb1b4b158ea048159f5d8d8c5294e7da3444edd76fbd903d7b7faa84e3382807a583d757325e1
+  containerd_version: v1.7.1
```

Comment:
let's skip this, we decided to go directly to v2

Reply:
hm, is there a fixed date for its release?